.github/actions/setup/action.yaml

```diff
@@ -26,8 +26,14 @@ runs:
     - name: Install project no MPI
       if: ${{ inputs.mpi == 'false' }}
       shell: sh
-      run: pip install ".[dev,aws]"
+      run: |
+        pip install noisepy-seis-io --no-deps --target tmp
```

Comment:
I worry this method is likely hiding an issue a user might face. It should be possible for the `noisepy-seis-io` package to be pulled in like any other dependency. E.g. we don't have to install any other packages like `numpy` or `pandas` like this,.